public/app/features/dashboard/containers/NewDashboardPage.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import { useEffectOnce } from 'react-use';
+
+import { config } from '@grafana/runtime';
+import { GrafanaRouteComponentProps } from 'app/core/navigation/types';
+import { loadDataSources } from 'app/features/datasources/state';
+import { useDispatch, useSelector } from 'app/types';
```

Comment:
Haven't noticed this before, but function import from types file feels funky. Probably not sth to fix here, but in a separate PR.